src/utils/utils_load_library.c

```diff
@@ -0,0 +1,72 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+#ifdef _WIN32
+
+// clang-format off
```

Comment:
add comment why we need this specyfic order of includes

Reply:
Done